### PR TITLE
having problems with my test file

### PR DESCRIPTION
**Pull request by FreemaGw**
having the problem as detailed in the email I sent you. Thanks for looking at it.

### CODE REVIEW

File: students/Ben_freeman/lesson03/tests/basic_operations_test.py [hunk 1]
```diff
@@ -0,0 +1,53 @@
+# pylint: disable=unused-import, unused-wildcard-import,
+# pylint: disable=too-few-public-methods, too-many-arguments, wildcard-import
+# pylint: disable=logging-format-interpolation, pointless-string-statement
+import sys
+import unittest
+sys.path.append("..")
```
Comment (ldconejo) [inline students/Ben_freeman/lesson03/tests/basic_operations_test.py:6]: Change to this:

`sys.path.append("../src/")`

File: students/Ben_freeman/lesson03/tests/basic_operations_test.py [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+from src import basic_operations as b_o
+from src import customers
```
Comment (ldconejo) [inline students/Ben_freeman/lesson03/tests/basic_operations_test.py:8]: Change to:

```
import basic_operations as b_o
import customers
```

**Comment by ldconejo** [comment]
Hi Ben,

See my comments on your code for a couple of changes to make.

Regards,

Luis

File: students/Ben_freeman/lesson03/tests/basic_operations_test.py (hunk 2 above)
Comment (FreemaGw) [reply students/Ben_freeman/lesson03/tests/basic_operations_test.py:8]: Pylint is still telling me that it cannot load customers or basic_operations even with the changes suggested.
bluescreenpylint

Comment (ldconejo) [reply students/Ben_freeman/lesson03/tests/basic_operations_test.py:8]: Update the code to your latest and open a new pull request for me to take a look.

-Luis

Comment (FreemaGw) [reply students/Ben_freeman/lesson03/tests/basic_operations_test.py:8]: sounds good, thanks!

**Pull request closed without merging**